docs/tutorials/ad-hoc-simulation/rosbag-replay-simulation.md

```diff
@@ -62,13 +62,17 @@
 
    ![after-autoware-launch](images/rosbag-replay/after-autoware-launch.png)
 
+   > ⚠️ It might happen that you see in the terminal some error and warning messages before playing the rosbag (next step) unit you play the rosbag and proper initialization occurs
```

Comment:
```suggestion
!!! warning

    It might happen that you see in the terminal some error and warning messages before playing the rosbag (next step) unit you play the rosbag and proper initialization occurs
```

Reply:
More info can be found at https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage